tools/metilene/metilene.xml

```diff
@@ -0,0 +1,158 @@
+        
+        <!-- Advanced parameters (folded) -->
+        <section name="Options" title="Advanced Options" expanded="False">
+	    <param
+                name="opt_M" 
+                label="The allowed nt distance between two CpGs within a DMR"
+                type="integer"
+	        optional="false"
+                value="300"
+	        help="-M, --maxdist"/>
+
+            <param
+                name="opt_m" 
+                label="The minimum # of CpGs in a DMR"
+                type="integer"
+	        optional="false"
+                value="10"
+	        help="-m, --mincpgs"/>
+            
+            <param
+                name="opt_d" 
+                label="The minimum mean methylation difference for calling DMRs"
+                type="float"
+	        optional="false"
+                value="0.1"
+	        help="-d, --minMethDiff"/>
+
+            <param
+                name="opt_v" 
+                label="Stringency of the valley filter (0.0 - 1.0)"
+                type="float"
+	        optional="false"
+                value="0.7"
+	        help="-v, --valley">
+                <validator type="in_range" min="0.0" max="1.0" 
+                           message="Valley filter value must be in [0,1]." />
+            </param>    
+	</section>
+    </inputs>
+    
+    <outputs>
+        <!-- finally: for bed-like use format=tabular -->
+        <data format="tabular" name="output_bedlike" 
+              label="${tool.name} on ${on_string}" />
+        <data format="bedgraph" name="output_bedgraph" 
+              label="${tool.name} on ${on_string}" />
+        <data format="bedgraph" name="output_qval_bedgraph" 
+              label="${tool.name} qval&lt;0.05 bedgraph on ${on_string}" />
+        <data format="pdf" name="output_qval_pdf" 
+              label="${tool.name} qval&lt;0.05 plots on ${on_string}" />
+        <data format="bedgraph" name="output_qval_out" 
+              label="${tool.name} qval&lt;0.05 out on ${on_string}" />
+    </outputs>
+    
+    <tests>
+        <test>
+	    <param name="opt_in1" value="example_S1.bedgraph,example_S2.bedgraph,example_S3.bedgraph,example_S4.bedgraph" />
+            <param name="opt_in2" value="example_S5.bedgraph,example_S6.bedgraph,example_S7.bedgraph,example_S8.bedgraph" />
+	    <output name="output_bedlike" file="metilene.out" />
+        </test>
+    </tests>
+    
+    
+    <help><![CDATA[
+**Fast and sensitive calling of differentially methylated regions
+from bisulfite sequencing data**
+
+Detailed documentation of the stand-alone tool metilene is available at
+.. __: http://www.bioinf.uni-leipzig.de/Software/metilene/manual.pdf
+
+The software metilene is available for download at
+.. __: http://www.bioinf.uni-leipzig.de/Software/metilene/
+    ]]></help>
+
+    <citations>
+        <citation type="bibtex">@Article{Juhling_Kretzmer_Bernhart-metil_Fast_and-2015,
+  author =	 {Juhling, Frank and Kretzmer, Helene and Bernhart, Stephan 
```

Comment:
If you like you can simply provide the doi like https://github.com/galaxyproject/tools-iuc/blob/master/tools/deseq2/deseq2.xml#L185